cg/services/order_validation_service/models/aliases.py

```diff
@@ -1,12 +1,7 @@
-from cg.services.order_validation_service.workflows.microsalt.models.sample import MicrosaltSample
 from cg.services.order_validation_service.workflows.mip_dna.models.case import MipDnaCase
 from cg.services.order_validation_service.workflows.mip_dna.models.sample import MipDnaSample
-from cg.services.order_validation_service.workflows.mutant.models.sample import MutantSample
 from cg.services.order_validation_service.workflows.tomte.models.case import TomteCase
 from cg.services.order_validation_service.workflows.tomte.models.sample import TomteSample
 
-
 SampleWithRelatives = TomteSample | MipDnaSample
 CaseContainingRelatives = TomteCase | MipDnaCase
-
-NonHumanSample = MutantSample | MicrosaltSample
```

Comment:
I agree that the name is a bit clunky but is there a reason for this being removed? I am pretty sure that this was added due to certain validation rules only being applicable for these workflows. If we change the type hints in the rules, will we not refer to properties which are not necessarily present in the sample?

Reply:
I misunderstood the inheritance model, I will put this back